optoforce/src/optoforce/optoforce.py

```diff
@@ -69,8 +70,13 @@ def __init__(self):
         """
         Initialize OptoforceDriver object
         """
-        port = rospy.get_param("~port", "/dev/ttyACM0")
-        self._serial = serial.Serial(port, 1000000, timeout=None)
+        try:
+            port = rospy.get_param("~port", "/dev/ttyACM0")
+            self._serial = serial.Serial(port, 1000000, timeout=None)
+        except serial.SerialException as e:
+            rospy.logfatal(e.message)
+            rospy.signal_shutdown("Serial connection failure")
```

Comment:
I am always a fan of making error messages as human readable as possible, but you are right, the info from e.message should be enough